src/styles/main.scss

```
@@ -1,3 +1,38 @@
 body {
   margin: 0;
+  height: 100vh;
+
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.snake {
+  height: 100%;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, auto));
+
+  font-size: 100px;
+  color: white;
+
+  @media (min-width: 900px) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  &__cell {
+    min-width: 300px;
+    min-height: 300px;
```

```suggestion
    height: 300px;
```
- Each block should be `300px` high.